test/Sema/accessibility_shared_private.swift

```diff
@@ -0,0 +1,76 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: %utils/split_file.py -o %t %s
+// RUN: %target-swift-frontend -swift-version 4 -typecheck %t/declarations.swift %t/other_file_extensions.swift -verify
```

Comment:
I'm struggling to find a clean way to do this. it looks like there's a `@-1` style syntax, that changes which run of FileCheck it matches, but I can't get it to work right or find any documentation on it. Any documentation other than the lit source?